### PR TITLE
Add AllowGuestUserShareToUsersNotInSiteCollection

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1934,6 +1935,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowGuestUserShareToUsersNotInSiteCollection 
+
+The AllowGuestUserShareToUsersNotInSiteCollection settings (defaulted to false) will allow or prevent guests from sharing to users not in the site collection
```
Comment (kaarins) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1940]: Is it multiple settings? It sounds like a single one.

**Comment by yogkumgit** [comment]
@tiburd Dario and Johan cannot be tagged or assigned/review, please help. Thanks

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@yogkumgit - Hi, Yogesh... For PowerShell PRs, it isn't necessary to remove the reviewers from the **Assignees** area. :) 

Thank you!

**Pull request merged**